third_party/move/move-model/bytecode/src/function_target_pipeline.rs

```diff
@@ -385,45 +399,61 @@ impl FunctionTargetPipeline {
         let rev_topo_order = Self::sort_in_reverse_topological_order(env, targets);
         info!("transforming bytecode");
         hook_before_pipeline(targets);
-        for (step_count, processor) in self.processors.iter().enumerate() {
-            if processor.is_single_run() {
-                processor.run(env, targets);
-            } else {
-                processor.initialize(env, targets);
-                for item in &rev_topo_order {
-                    match item {
-                        Either::Left(fid) => {
-                            let func_env = env.get_function(*fid);
-                            targets.process(&func_env, processor.as_ref(), None);
-                        },
-                        Either::Right(scc) => 'fixedpoint: loop {
-                            let scc_env: Vec<_> =
-                                scc.iter().map(|fid| env.get_function(*fid)).collect();
-                            for fid in scc {
+        let max_loop = self.max_loop.max(1);
```

Comment:
As I said in the previous review, this code should not be changed. If you need a fixedpoint loop please do this locally to the decompiler. 

The functionality to have a global fixedpoint loop on transformed code in the pipeline seems to be very unusual and as never been needed in the years this code has been used in many different applications.

If we really would like a fixedpoint loop on bytecode change we would not implement it like this, but as a combinator `loop_until_no_change(Processor) -> Processor`.